backend-api/template.yaml

```diff
@@ -1113,9 +1156,14 @@ Resources:
         - !Ref AWS::NoValue
 
 Outputs:
-  PublicApiUrl:
-    Description: Public API Gateway base URL
-    Value: !Sub https://${BackendAsyncCredentialPublicApi}.execute-api.${AWS::Region}.amazonaws.com/${Environment}/
+  SessionsApiUrl:
```

Comment:
Out of curiosity, would this work instead? 🤔 

`Value: !Sub https://${SessionsApiDomainName}`

Reply:
It should do yes - I'll raise a follow-up PR

Reply:
I just thought it might be nicer but it doesn't have to change